proxy/coco/src/peer/run_state.rs

```diff
@@ -254,165 +275,77 @@ impl RunState {
         }
     }
 
-    fn handle_peer_event(event: PeerEvent) -> Vec<Command> {
-        match event {
-            PeerEvent::GossipFetch(FetchInfo {
-                result: PutResult::Applied(Gossip { urn, .. }),
-                ..
-            }) => vec![Command::Include(urn)],
-            PeerEvent::GossipFetch(_) => vec![],
-        }
-    }
-
     /// Handle [`input::Sync`]s.
     fn handle_peer_sync(&mut self, input: &input::Sync) -> Vec<Command> {
-        if let Status::Syncing { synced, syncs } = self.status {
+        if let Status::Syncing {
```

Comment:
The logic is changing here because of the move from `usize` to `HashSet<PeerId>`